packages/zwave-js/src/lib/test/driver/s2FailureDisablesInterview.test.ts

```diff
@@ -0,0 +1,60 @@
+import { MockController } from "@zwave-js/testing";
+import { test as baseTest } from "vitest";
+import { createDefaultMockControllerBehaviors } from "../../../Testing.js";
+import type { Driver } from "../../driver/Driver.js";
+import { createAndStartTestingDriver } from "../../driver/DriverMock.js";
+import { ZWaveNode } from "../../node/Node.js";
+
+interface LocalTestContext {
+	context: {
+		driver: Driver;
+		controller: MockController;
+	};
+}
+
+const test = baseTest.extend<LocalTestContext>({
+	context: [
+		async ({}, use) => {
+			// Setup
+			const context = {} as LocalTestContext["context"];
+
+			const { driver } = await createAndStartTestingDriver({
+				skipNodeInterview: true,
+				loadConfiguration: false,
+				beforeStartup(mockPort, serial) {
+					const controller = new MockController({
+						mockPort,
+						serial,
+					});
+					controller.defineBehavior(
+						...createDefaultMockControllerBehaviors(),
+					);
+					context.controller = controller;
+				},
+			});
+			context.driver = driver;
+
+			// Run tests
+			await use(context);
+
+			// Teardown
+			driver.removeAllListeners();
+			await driver.destroy();
+		},
+		{ auto: true },
+	],
+});
+
+test("ensure cached skipInterview value", ({ context, expect }) => {
+	const { driver } = context;
+
+	const oldNode = new ZWaveNode(2, driver);
+	expect(oldNode.skipInterview).toBe(false);
+	oldNode.skipInterview = true;
+	expect(oldNode.skipInterview).toBe(true);
+
+	const newNode = new ZWaveNode(2, driver);
+	expect(newNode.skipInterview).toBe(true);
+});
+
+// TODO: Make the node fail S2 and then check if skipInterview is true and also ensure that "interview completed" is NOT emitted
```

Comment:
A test for this is going to be challenging, especially if you don't have too much experience with the low level Z-Wave details.
The correct approach would be an `integrationTest(...)` based test, but you'd have to define the end device behavior up to a certain point.

I'd leave this out for now - I'll test manually with an actual device and maybe write a test if there is a need for it.